docs/src/dev/env.md

```diff
@@ -13,6 +13,18 @@ testing only. It is enabled whenever the `tests` crate is being compiled, so a
 regular `cargo build` in the root of the workspace will enable the feature (to
 build *without* the feature enabled, do `cargo build -p ykcapi`).
 
+## Build-time Variables
+
+### `YKB_YKLLVM_BIN_DIR`
+
+Under normal circumstances, yk builds a copy of its LLVM fork "ykllvm" and uses
+it to build interpreters. You can use your own ykllvm build by specifying the
```

Comment:
> and uses it to build interpreters

I'm not sure about this bit. Which interpreters do you mean here? yklua and ykcbf?

Reply:
This repo doesn't build those interpreters, so I think we should remove that part.

Reply:
Er, actually, that's kind of a good point. At the moment it's a mess: we expect the user to use the *correct* `clang` for ykllvm. Arguable we should have `yk-config --cc` which gives you the path to our `clang` so you can do something like this in a Makefile:

```
CC=`yk-config --cc`
```

What do you think?

Reply:
I suspect (but am not 100% sure) I can do this in the PR I'm not working on.

I suggest we keep this documentation as-is, because it's sort-of true now, and we can make it better in the next PR.

Reply:
+1. That works for me